assets/node_modules/@enhavo/app/action/ActionInterface.ts

```diff
@@ -3,4 +3,6 @@ import { ComponentAwareInterface } from "@enhavo/core/index";
 export default interface ActionInterface extends ComponentAwareInterface
 {
     execute(): void;
+
+    subActions(): ActionInterface[];
```

Comment:
`subActions` is only for a few action and shouldn't be added to the `ActionInterface`